stubs/logger.pyi

```diff
@@ -0,0 +1,3 @@
+from logging import Logger
+
+def setup_logger(name: str) -> Logger: ...  # noqa: E302
```

Comment:
Interesting, is this stub really necessary? `logging` is part of the standard library.

Reply:
[2b2b94a](https://github.com/ethereum/execution-spec-tests/pull/461/commits/2b2b94a8de546dd91e2fc1a805ec1ac9e2baad1d)